go.sum

```diff
@@ -166,8 +166,12 @@ github.com/confluentinc/go-editor v0.11.0 h1:fcEALYHj7xV/fRSp54/IHi2DS4GlZMJWVgr
 github.com/confluentinc/go-editor v0.11.0/go.mod h1:nEjwqdqx8S7ZGjXsDvRgawsA04Fu2P/KAtA8fa5afMI=
 github.com/confluentinc/go-netrc v0.0.0-20220321173724-4d50f36ff450 h1:YJq9ZRhj049uo3hX7V8imJid+FJEBFyRmVcdruhU3y4=
 github.com/confluentinc/go-netrc v0.0.0-20220321173724-4d50f36ff450/go.mod h1:GGVB1yDj9YYQTxoo5vTRGWuQKccGN00bvyit4O5jznI=
-github.com/confluentinc/go-prompt v0.2.26 h1:DuzCAzU2dQoka8dJ0TiNDHXEbgIN5iVMZCLbSdtZgD4=
-github.com/confluentinc/go-prompt v0.2.26/go.mod h1:zevvJDLYjJBw/CFOd+9O1xBChio6z3WP7skZbw0YDC4=
+github.com/confluentinc/go-prompt v0.2.27-0.20240213181507-604017e2bb08 h1:Q9sQIhMeRxlNZaIaBBt21dyjzpNfIwaUxoYmVZVF828=
+github.com/confluentinc/go-prompt v0.2.27-0.20240213181507-604017e2bb08/go.mod h1:zevvJDLYjJBw/CFOd+9O1xBChio6z3WP7skZbw0YDC4=
+github.com/confluentinc/go-prompt v0.2.27 h1:OfodwY4NIYaF6aJ0W7S2vblijwZ7RJ+CkqEy5hDcoSY=
```

Comment:
Wonder why 0.2.27 is still here. Run `go mod tidy`?

Reply:
oh yeah forgot that

Reply:
done